lib/matplotlib/font_manager.py

```diff
@@ -1117,12 +1118,11 @@ def score_family(self, families, family2):
                 options = [x.lower() for x in options]
                 if family2 in options:
                     idx = options.index(family2)
-                    return ((0.1 * (idx / len(options))) *
-                            ((i + 1) / len(families)))
+                    return (i + (idx / len(options))) * step
```

Comment:
This is where I get confused, and either don't understand or don't agree with the scoring algorithm. If it matches an alias, and that alias was provided first in the font families list, and the specific font being scored appears first in the alias list, wouldn't that score a 0? How could there be a better match?